modules/runtimes/PrefabPython.py

```diff
@@ -70,9 +70,8 @@ def build(self, jumpscale_branch='development', include_jumpscale=True, reset=Fa
                 C = self.replace(C)
 
             else:
-
+                self.prefab.core.run('apt-get install zlib1g-dev libncurses5-dev libbz2-dev liblzma-dev libsqlite3-dev libreadline-dev libssl-dev libsnappy-dev -y')
```

Comment:
Now your just working around instead of trying to find why ` self.prefab.system.package.install` didn't install anything.
Instead investigate why ` self.prefab.system.package.install` didn't work and fix that root cause.

Reply:
this is a temporary solution only. As stated. It's to unblock further development till find the root cause

Reply:
is this method linux only ? cause now since we use apt-get explicitly that will only work on ubuntu